Sources/Core/Structure/SECP256k1.swift

```diff
@@ -336,6 +336,10 @@ extension SECP256K1 {
     }
 
     internal static func randomBytes(length: Int) -> Data? {
+        #if os(Linux)
+//        return Data(URandom().bytes(count: length))
+        return try? Data.random(length: length)
+        #else
```

Comment:
Seems like it's good time to change this sepc256k implementation to those that bitcoin maintains, that did @JeneaVranceanu proposed in the past. I'd bet it's cross platform.

Reply:
It should be cross-platform as it's written in C but it's not a solution here as the code you are commenting on is our custom extension function and it's just the question of how to get secure random bytes on Linux.

I'd not complicate this PR with the secp256k1 update.